tests/foreman/ui/test_oscapcontent.py

```diff
@@ -76,27 +73,6 @@ def test_negative_create_with_invalid_name(self):
                     self.assertIsNotNone(session.nav.wait_until_element(
                         common_locators['haserror']))
 
-    @tier1
-    def test_positive_default(self):
```

Comment:
We still need this test-case. 
1) For sat6.2 steps have changed and we need to run a command "foreman-rake foreman_openscap:bulk_upload:default"
2) I have raised a bug and requested this to be part of the installer, https://bugzilla.redhat.com/show_bug.cgi?id=1336374

We have 2 options:
a) update automation-tools to run "foreman-rake foreman_openscap:bulk_upload:default"
b) add bugzilla check for the testcase, 1336374